src/app/app.module.ts

```diff
@@ -5,10 +5,24 @@ import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { NoopAnimationsModule } from '@angular/platform-browser/animations';
 import { MatCardModule } from '@angular/material/card';
+import { TranslateLoader, TranslateModule } from "@ngx-translate/core";
+import { PomTranslateLoader, Language } from "./core/translate-loader.service";
 
 @NgModule({
   declarations: [AppComponent],
-  imports: [BrowserModule, AppRoutingModule, NoopAnimationsModule, MatCardModule],
+  imports: [
+    BrowserModule,
+    AppRoutingModule,
+    NoopAnimationsModule,
+    MatCardModule,
+    TranslateModule.forRoot({
+      defaultLanguage: Language.pl_PL,
```

Comment:
Tak się zastanawiam czy jednak domyślny nie powinien być angielski?

Pisząc html chyba robimy np. `<label i18n>Type of Support</label>`.

Reply:
Przykład użycia

{{ 'WHAT_YOU_LOOK_FOR' | translate }}

No dla tych co ogłaszają pomoc domyślny polski,
Dla tych co szukają domyślny Ukraiński

Wydaje się najrozsądniejsze.

Reply:
Zostałbym przy polskim, widać dobrze opcje wyboru języka.

Reply:
potwierdzone, polski somyślny